lib/cli/parse.rs

```diff
@@ -1492,4 +1526,38 @@ mod tests {
             }
         }
     }
+    mod account_identifier{
```

Comment:
Nit: we'd normally have a blank line before a new module like this, and also before each test case.